prepare transaction/eosJS/holacontrato.js

```diff
@@ -0,0 +1,45 @@
+
+const { eosioConfig } = require('config')
```

Comment:
you need to import eosjs

Reply:
@AngeloCG97  please see these examples:

https://developers.eos.io/manuals/eosjs/latest/transaction-examples